doc/source/groupby.rst

```diff
@@ -1014,6 +1014,22 @@ Regroup columns of a DataFrame according to their sum, and sum the aggregated on
    df
    df.groupby(df.sum(), axis=1).sum()
 
+Groupby by Indexer to 'resample' data.
+
+Resampling produces new hypothetical samples(resamples) from already existing observed data or from a model that generates data. These new samples are similar to the pre-existing samples.
+
+In order to resample to work on indices that are non-datetimelike , the following procedure can be utilized.
+
+In the following examples, **df.index // 5** returns a binary array which is used to determine what get's selected for the groupby operation.
+
+.. note:: The below example shows how we can downsample which is  the throwing away of samples. Here by using **df.index // 5**, we are aggregating the samples in bins. By applying **std()** function, we aggregate the information contained in many samples into a small subset of values which is their standard deviation thereby reducing the number of samples.
```

Comment:
I still don't like the "throwing away of samples". You don't throw them away, IMO you process them in some way (eg taking the mean of each group of samples)